cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -4,13 +4,19 @@ kind: Template
 metadata:
```

Comment:
What is the meaning of "cluster-service-namespace" in this context?

Reply:
I did not add this, maybe @geoberle can comment on this.